diff --git a/tests/src/Kernel/PermanentDatabaseBackendTest.php b/tests/src/Kernel/PermanentDatabaseBackendTest.php
index e0d2a75..233056b 100644
--- a/tests/src/Kernel/PermanentDatabaseBackendTest.php
+++ b/tests/src/Kernel/PermanentDatabaseBackendTest.php
@@ -36,7 +36,7 @@ class PermanentDatabaseBackendTest extends GenericCacheBackendUnitTestBase {
   /**
    * Test Drupal\Core\Cache\CacheBackendInterface::deleteAll().
    */
-  public function testDeleteAll() {
+  public function testDeleteAll(): void {
     $backend_a = $this->getCacheBackend();
     $backend_b = $this->getCacheBackend('bootstrap');
 
@@ -56,7 +56,7 @@ class PermanentDatabaseBackendTest extends GenericCacheBackendUnitTestBase {
   /**
    * Testing the basic goals of the permanent cache.
    */
-  public function testSetGet() {
+  public function testSetGet(): void {
     $backend = $this->getCacheBackend();
     $cid = 'test';
     $cache_value = 'This does not matter.';
